test/org/jetbrains/plugins/ideavim/action/CopyActionTest.java

```diff
@@ -146,6 +146,8 @@ public void testStateAfterYankVisualBlock() {
   }
 
   // VIM-476 |yy| |'clipboard'|
+  // TODO: Review this test
+  // This doesn't use the system clipboard, but the TestClipboardModel
```

Comment:
Testing the output of `:registers` requires mocking the system clipboard to get stable results. That changes the purpose of these tests. Previously, they tested:

a) the `*` register is the system clipboard register
b) integration with the system clipboard

Now, they simply test a) the `*` register is the system clipboard register, but the data from the clipboard comes from a mock. 

If this is ok, I'll remove the TODO comments. If we also want to test b) then we'll need new tests.